src/compiler/types.ts

```diff
@@ -1493,6 +1499,8 @@ module ts {
     // Class and interface types (TypeFlags.Class and TypeFlags.Interface)
     export interface InterfaceType extends ObjectType {
         typeParameters: TypeParameter[];           // Type parameters (undefined if non-generic)
+        outerTypeParameters: TypeParameter[];      // Outer type parameters (undefined if none)
+        localTypeParameters: TypeParameter[];      // Local type parameters (undefined if none)
     }
 
     export interface InterfaceTypeWithBaseTypes extends InterfaceType {

```

Comment:
1. I meant in this PR, but I guess it's been changed before.
2. To clarify, the export modifier in a statement context is not *allowed*, it's just parsed as a statement.